Fix CI Drift for Integration tests

as per https://github.com/abcxyz/lumberjack/pull/14 

```
--- FAIL: TestEndpoints (0.00s)
    --- FAIL: TestEndpoints/https://ci-with-server-17096-go-zknpj76j4a-uc.a.run.app (35.13s)
        main_test.go:80: Generated UUID: 99980fc9-7a7d-4fa4-bf68-8d6d39902910.
        main_test.go:132: Matching entry not found, retrying...
        main_test.go:135: Query error: googleapi: Error 404: Not found: Table github-ci-server:audit_logs.auditlog_gcloudsolutions_dev_data_access was not found in location us-central1, notFound.
        main_test.go:132: Matching entry not found, retrying...
        main_test.go:135: Query error: googleapi: Error 404: Not found: Table github-ci-server:audit_logs.auditlog_gcloudsolutions_dev_data_access was not found in location us-central1, notFound.
        main_test.go:132: Matching entry not found, retrying...
        main_test.go:135: Query error: googleapi: Error 404: Not found: Table github-ci-server:audit_logs.auditlog_gcloudsolutions_dev_data_access was not found in location us-central1, notFound.
        main_test.go:132: Matching entry not found, retrying...
        main_test.go:135: Query error: googleapi: Error 404: Not found: Table github-ci-server:audit_logs.auditlog_gcloudsolutions_dev_data_access was not found in location us-central1, notFound.
        main_test.go:132: Matching entry not found, retrying...
        main_test.go:135: Query error: googleapi: Error 404: Not found: Table github-ci-server:audit_logs.auditlog_gcloudsolutions_dev_data_access was not found in location us-central1, notFound.
        main_test.go:132: Matching entry not found, retrying...
        main_test.go:139: Retry failed: retryable: audit log not found.
    --- FAIL: TestEndpoints/https://ci-with-server-17096-java-zknpj76j4a-uc.a.run.app (38.68s)
        main_test.go:80: Generated UUID: ef54cb76-a4ec-4ffe-819b-88f2a6aae36b.
        main_test.go:132: Matching entry not found, retrying...
        main_test.go:135: Query error: googleapi: Error 404: Not found: Table github-ci-server:audit_logs.auditlog_gcloudsolutions_dev_data_access was not found in location us-central1, notFound.
        main_test.go:132: Matching entry not found, retrying...
        main_test.go:135: Query error: googleapi: Error 404: Not found: Table github-ci-server:audit_logs.auditlog_gcloudsolutions_dev_data_access was not found in location us-central1, notFound.
        main_test.go:132: Matching entry not found, retrying...
        main_test.go:135: Query error: googleapi: Error 404: Not found: Table github-ci-server:audit_logs.auditlog_gcloudsolutions_dev_data_access was not found in location us-central1, notFound.
        main_test.go:132: Matching entry not found, retrying...
        main_test.go:135: Query error: googleapi: Error 404: Not found: Table github-ci-server:audit_logs.auditlog_gcloudsolutions_dev_data_access was not found in location us-central1, notFound.
        main_test.go:132: Matching entry not found, retrying...
        main_test.go:132: Matching entry not found, retrying...
        main_test.go:139: Retry failed: retryable: audit log not found.
FAIL
FAIL	github.com/abcxyz/lumberjack/integration/httptestrunner	38.686s
FAIL
```

--- a/scripts/integration_build.sh
+++ b/scripts/integration_build.sh
@@ -28,7 +28,7 @@ SHELL_APP_PROJECT_ID=github-ci-app-0
 BACKEND_PROJECT_ID=github-ci-server
 BIGQUERY_DATASET_ID=audit_logs
 
-CI_SERVICE_ACCOUNT=github-access-sa@lumberjack-dev-infra.iam.gserviceaccount.com
+CI_SERVICE_ACCOUNT=gh-access-sa@lumberjack-dev-infra.iam.gserviceaccount.com
 GCLOUD_ACCOUNT=$(gcloud config get-value account)
 if [[ $GCLOUD_ACCOUNT == $CI_SERVICE_ACCOUNT ]]; then
   # When running in CI, impersonate the service account to have its email included in the ID token.

